package.json

```diff
@@ -51,7 +51,7 @@
     "@types/tar-stream": "^2.2.2",
     "@typescript-eslint/eslint-plugin": "^5.61.0",
     "@typescript-eslint/parser": "^5.61.0",
-    "eslint": "^8.44.0",
+    "eslint": "^8.45.0",
```

Comment:
Please revert the changes done to package-lock.json and package.json

Reply:
reverted